src/courses/tests/test_models.py

```diff
@@ -323,6 +323,12 @@ def test_xp_per_day_ave(self, days_so_far):
         xp_per_day = self.course_student.xp_per_day_ave()
         self.assertEqual(xp_per_day, 0)
 
+    def test_cascade_protect(self):
+        """ 
+            quick test to see if cascade
+        """ 
+        self.assertRaises(Exception, self.course.delete)
+
```

Comment:
This is in the wrong test class (CourseStudent instead of Course, and the docstring text appears to be cutoff.

Reply:
Moved the test to CourseModelTest